qcodes_contrib_drivers/drivers/ERAInstruments/erasynth.py

```diff
@@ -0,0 +1,1014 @@
+"""Driver for the ERASynth/ERASynth+/ERASynth++ signal generators.
+
+Author: Victor Negîrneac, vnegirneac@qblox.com
+
+For official instrument support visit:
+
+- https://erainstruments.com
+- https://github.com/erainstruments/
+
+NB this driver adds parameters using the new `@add_paramter` decorator-style.
+See QCoDeS docs on writing drivers for more details.
+"""
+from __future__ import annotations
+
+from typing import Union
+import time
+import json
+from qcodes import Instrument
+from qcodes import validators
+from qcodes.instrument.base import InstanceAttr, add_parameter
```

Comment:
again, suggest to guard this with an import error, and inform the users that they need to have qcodes > X version installed.